src/main/cli/cli.c

```diff
@@ -3556,6 +3556,43 @@ static void cliGpsPassthrough(const char *cmdName, char *cmdline)
 
     gpsEnablePassthrough(cliPort);
 }
+
+static void cliGpsInfo(const char *cmdName, char *cmdLine) {
+    UNUSED(cmdName);
+    UNUSED(cmdLine);
+
+    cliPrintLinef("# MON-VER acquired: %s", gpsData.acquiredMonVer ? "true" : "false");
+
+    if (gpsData.acquiredMonVer) {
+
+        cliPrintLinef("swVersion: ");
+        cliPrint(" ");
+        cliPrintLine(&gpsData.monVer.swVersion[0]);
+
+        cliPrintLinef("hwVersion: ");
+        cliPrint(" ");
+        cliPrint(&gpsData.monVer.hwVersion[0]);
+        cliPrint(" (");
+        cliPrint(ubloxVersion_str[gpsData.unitVersion]);
+        cliPrintLine(")");
+
+        cliPrintLinef("extension: ");
+
+        for (size_t i = 0; i < 10; ++i) {
+            cliPrint(" ");
+            for(size_t j = i * 30; j < i * 30 + 30; ++j) {
+                if (j == i * 30 && gpsData.monVer.extension[j] == 0) {
+                    return;
+                }
+                if (gpsData.monVer.extension[j] == 0) {
+                    cliPrintLinefeed();
+                    break;
+                }
+                cliPrintf("%c", gpsData.monVer.extension[j]);
+            }
```

Comment:
```suggestion
        const char (*extension)[30] = (const char(*)[30])&gpsData.monVer.extension;
        for (size_t i = 0; i < 10; ++i) {
            cliPrint(" ");
            if (extension[i][0] == '\0')
                return;     // break is fine here, also consider if space is necessary
            for(size_t j = 0; j < 30 && extension[i][j] != '\0'; ++j) {
                cliPrintf("%c", extension[i][j]);
            }
            cliPrintLinefeed();
         }
```

Reply:
can you explain the difference to my solution?

Reply:
It should be functionally equivalent, but possibly easier to read